samples/testbot/test-bot/BotOrchestration.cs

```diff
@@ -0,0 +1,379 @@
+                //                                  End date must be 15 minutes less than the current time
+                //                                  Start date time must be 30 minutes less than current date time
+
+                DateTime endTimeUtc = context.CurrentUtcDateTime.AddMinutes(5);
+                if (endTimeUtc.Subtract(startTimeUtc).TotalMinutes < 15)
+                {
+                    startTimeUtc = endTimeUtc.AddMinutes(-15);
+                }
+
+                string startTime = startTimeUtc.ToString("yyyy-MM-ddTHH:mm");
+                string endTime = endTimeUtc.ToString("yyyy-MM-ddTHH:mm");
+                string resultsAvailableTime = (endTimeUtc.AddMinutes(15)).ToString("yyyy-MM-dd HH:mm UTC");
+
+                string subscriptionId = testParameters.SubscriptionId;
+                string resourceGroup = testParameters.ResourceGroup;
+                string appName = testParameters.AppName;
+                string detectorName = testParameters.DetectorName;
+
+                string link = $"https://applens.azurewebsites.net/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Web/sites/{appName}/detectors/{detectorName}?startTime={startTime}&endTime={endTime}";
+
+                string appLensLinkString = $"You can view more detailed results in [AppLens]({link}) (Microsoft internal). The results will be available at {resultsAvailableTime}🔗📈";
+                finalMessage += Environment.NewLine + Environment.NewLine + appLensLinkString;
+                await context.CallActivityAsync(nameof(PatchGitHubComment), (testParameters.GitHubCommentIdApiUrl, finalMessage));
+
+                /*
+                // TODO: resource group clean up functionality
+                TimeSpan cleanupInterval = TimeSpan.FromHours(1);
+                DateTime cleanupTime = context.CurrentUtcDateTime.Add(cleanupInterval);
+                finalMessage += Environment.NewLine + Environment.NewLine + $"The test app **{appName}** will be deleted at {cleanupTime:yyyy-MM-dd hh:mm:ss} UTC.";
+
+                await context.CallActivityAsync(nameof(PostGitHubComment), (testParameters.GitHubCommentApiUrl, finalMessage));
+
+                log.LogInformation($"Sleeping until {cleanupTime:yyyy-MM-dd hh:mm:ss} UTC to delete the test function app.");
+                await context.CreateTimer(cleanupTime, CancellationToken.None);
+                */
+            }
+            catch (Exception)
+            {
+                string message = $"An unexpected failure occurred! 💣 Unfortunately, we can't continue the test run and the test app will be deleted. 😞";
+                await context.CallActivityAsync(nameof(PatchGitHubComment), (testParameters.GitHubCommentIdApiUrl, message));
+                throw;
+            }
+            finally
+            {
+                /*   
+                // TODO: add functionality to delete the resource group when the PR is merged or closed
```

Comment:
Is this in scope of this PR still? If not, let's open an issue to add this later.